### DISCUSSION TITLE
Code contribution

### DISCUSSION THREAD

**Discussion by chadmhicks** [shap/shap#3693] [Ideas] [answered]
For a project at work, I have added Plotly support to the following plots: bar, decision, heatmap, scatter and beeswarm.  I have permission and would like to contribute these changes back to the SHAP repository.  What is your process for contributing code back to the repository?

**Comment by connortann** [comment, marked as answer]
Hi - please see our Contributing Guide here:

https://shap.readthedocs.io/en/latest/contributing.html
